client/src/app/shared/shared-share-modal/video-share.component.ts

```diff
@@ -152,10 +154,10 @@ export class VideoShareComponent {
     )
   }
 
-  async getVideoIframeCode () {
+  async getVideoEmbedCode (responsive = false) {
```

Comment:
You can access this value in `this.customizations.response`, no need to send it as a param :)

Reply:
Yes but it will broke the embed preview. I mainly did it to ensure that the preview does not get the responsive feature, so maybe I can change this param to `isPreview` ?

Reply:
Okay then prefer `options: { responsive: boolean }` so it's clearer when you read the function call (no need to check what is the purpose of the `true` or `false` as arguments